x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/index.test.ts

```diff
@@ -916,9 +915,9 @@ describe('Observability AI Assistant client', () => {
               last_updated: expect.any(String),
               title: 'My predefined title',
               token_count: {
-                completion: 24,
-                prompt: 458,
-                total: 482,
+                completion: expect.any(Number),
```

Comment:
Why this change?

Reply:
These numbers are from the default system message and functions etc that we put into the Assistant, so every time we change something, we have to update these numbers. I think a simple check for it being a number is good enough.

Reply:
Yeah agreed, it's just very painful to update now. Maybe we can have some other kind of mechanism.